### PR TITLE
[baseline] fixes #1998 missing super type cuts off inheritance of java.lang.Object

### LINKED ISSUE #1998: missing super type in baseline cuts off inheritance of java.lang.Object
When a class inherits from a super type which is missing during baseline, the baseline forgets that every class inherits from `java.lang.Object` and this can result in erroneous removed method outcomes in baseline.

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
just waiting for result from OSGi build.

File: biz.aQute.bndlib.tests/src/test/baseline/BaselineTest.java [hunk 1]
```diff
@@ -555,4 +557,51 @@ public void testMovePackageToBundleClassPath() throws Exception {
...
+	// baseline would _forget_ that every class inherits from Object, and _lose_
+	// Object's methods if not directly implemented.
+	public void testCutOffInheritance() throws Exception {
+		Processor processor = new Processor();
+
+		DiffPluginImpl differ = new DiffPluginImpl();
+		Baseline baseline = new Baseline(processor, differ);
+
+		Jar older = new Jar(IO.getFile("jar/baseline/inheritance-change-1.0.0.jar"));
+		Jar newer = new Jar(IO.getFile("jar/baseline/inheritance-change-1.1.0.jar"));
```
Comment (bjhargrave) [inline biz.aQute.bndlib.tests/src/test/baseline/BaselineTest.java]: You need to put the Jar constuctors in a try-with-resource to properly close the files.

`try (Jar older = ...; Jar newer = ...) {`

Comment (rotty3000) [inline biz.aQute.bndlib.tests/src/test/baseline/BaselineTest.java]: you want me to change all the tests?

Comment (rotty3000) [inline biz.aQute.bndlib.tests/src/test/baseline/BaselineTest.java]: better? See additional commit.

**Pull request merged**